static/js/components/inputs/CountrySelectField.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import _ from 'lodash';
+import iso3166 from 'iso-3166-2';
+import SelectField from './SelectField';
+
+let countryOptions = Object.keys(iso3166.data).map(code => ({
+  value: code,
+  label: iso3166.data[code]['name']
+}));
+countryOptions = _.sortBy(countryOptions, 'label');
```

Comment:
we only need to create this set of country options once. any instance of CountrySelectField is going to use the exact same set of options, so to me it made sense to avoid calculating the options every time a CountrySelectField renders